Gruntfile.js

```diff
@@ -54,7 +54,17 @@ module.exports = function(grunt) {
         options: {
           jshintrc: '.jshintrc'
         },
-}
+      },
+
+      concat: {
+          options: {
+            separator: ';',
+          },
+          dist: {
+            src: ['node_modules/jquery/dist/jquery.min.js', 'node_modules/leaflet/dist/leaflet.js', 'src/object.js'],
```

Comment:
Take a look at the Browserify example in PublicLab.Editor -- it'll be helpful! or in https://github.com/publiclab/inline-markdown-editor/blob/master/Gruntfile.js#L55 -- a bit simpler!